src/App.js

```diff
@@ -1,11 +1,12 @@
 import {useState} from 'react';
+import {Routes, Route} from 'react-router-dom';
 import styled from 'styled-components';
 
-import ChairCard from './component/ChairCard/ChairCard';
-import data from './data.json';
+import ChairList from './component/ChairList/ChairList';
+import chairsData from './data.json';
+import Home from './pages/Home';
 
 export default function App() {
-  const chairsData = data;
   const [showDescription, setShowDescription] = useState(false);
 
   const toggleDescription = id => {
```

Comment:
What is `id`? `showDescription` seems to be a toggle state, either `true` or `false`. When you update this state with the `id` your are assigning another value than `true / false` which might be confusing for other developers who look at your code. 

What do you want to achieve with this state and function?

EDIT:

From looking at your code, I would suggest to remove the state from the App since it is not global. Add the state to your `ChairCard` component and just write:
```js
function toggleDescription() {
  setShowDescription(!showDescription);
}
```
This way the state is within your component and you don't have to add an `id` and check it all the time.

Reply:
Ah, okay, dann ergibt deine Lösung Sinn. Aus dem Kopf heraus fällt mir dazu keine Lösung ein 🤔. Sonst lass es erstmal so, wie du es hattest und füge ggf. einen Kommentar oder so hinzu? Sorry für das hin und her 😅